### PR TITLE
Laravel 6: Misc changes

### PR DESCRIPTION
**Pull request by eronisko**
Other, minor changes (see commits)

### CODE REVIEW

File: resources/views/dielo.blade.php [hunk 1]
```diff
@@ -481,7 +481,7 @@ class="btn btn-default btn-outline uppercase sans">{{ trans('general.close') }}<
 
 {{-- Report item details to Google Tag Manager --}}
 <script defer>
-    dataLayer.push({{ json_encode([
+    dataLayer.push({!! json_encode([
```
Comment (rastislav-chynoransky) [inline resources/views/dielo.blade.php:484]: maybe use `@json` directive instead? https://laravel.com/docs/6.x/blade#displaying-data

**Review by rastislav-chynoransky — APPROVED** [review]
LGTM! I'm just curious how did missing `$keyType` affect behavior of eloquent models?

**Comment by eronisko** [comment]
@rastislav-chynoransky https://github.com/laravel/framework/pull/28153 this is the change that required `$keyType`. But not having it didn't seem to break things in my local env.

**Comment by rastislav-chynoransky** [comment]
@eronisko thanks!

File: resources/views/dielo.blade.php (hunk 1 above)
Comment (eronisko) [reply resources/views/dielo.blade.php:484]: I will! Great suggestion, thanks.

**Pull request merged**